operate/webapp/src/main/java/io/camunda/operate/webapp/security/permission/PermissionsService.java

```diff
@@ -136,6 +136,24 @@ private boolean hasPermissionForResource(
     return isAuthorizedFor(resourceId, resourceType, permissionType);
   }
 
+  /**
+   * hasPermissionForResource
+   *
+   * @return true if the user has the given permission for the resource
+   */
+  private boolean hasPermissionForResource(
```

Comment:
🔧 Could we rename this methdo to `hasPermissionResourceType`? Mainly because we have another method in this class which is already called `hasPermissionForResource` which checks the permissions for the `AuthorizationResourceType.RESOURCE`. It's bit less confusing if we don't overload it.